cmd/experimental/kjobctl/pkg/builder/builder.go

```diff
@@ -313,6 +313,17 @@ func (b *Builder) validateGeneral(ctx context.Context) error {
 		}
 	}
 
+	// priority class is checked excluding interactive mode
+	if b.modeName != v1alpha1.InteractiveMode {
```

Comment:
I think it's not necessary. Because we are already checked it on the command and will skip if we don't need.